config/locales/en.yml

```diff
@@ -102,6 +102,7 @@ en:
       welcome_message: Welcome to RubyGems.org! Click the link below to verify your email.
     email_reset:
       visit_link_instructions: You changed your email address on RubyGems.org. Please visit the following url to re-activate your account.
+      link_expiration_explanation: Please keep in mind that this link is only valid for 15 minutes
```

Comment:
Wha about `Please keep in mind that this link is only valid for 15 minutes.` (last character added)